content/en/blog/_posts/2024-04-22-userns-beta/index.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: blog
+title: "Kubernetes 1.30: User Namespaces hits Beta!"
```

Comment:
How about:
```suggestion
title: "Kubernetes 1.30: User Namespaces Hit Beta!"
```

(titles in Title Case)

or

```suggestion
title: "Kubernetes 1.30: Beta Support For Pods With User Namespaces"
```

Reply:
Fixed.